.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-1.9.3
```

Comment:
@celsocrivelaro  I'm completely in favor to drop this support, but we have many internal projects using 1.9.3. Drop the support of heartcheck for this platform can be a option.